pkgs/by-name/pf/pflotran/package.nix

```
@@ -14,6 +14,25 @@
   python3,
 }:
 
+let
+  /*
+    Upstream petsc has lots of fortran api change since 3.22.*
+    We will keep using old petsc-3.21.4 until pflotran support latest petsc.
+    Pflotran also requires Parmetis support in Petsc to have actual parmetis support.
+  */
+  myPetsc =
```

```suggestion
  petsc' =
```
lets not use tutorial variable names

---

Haha, fair :rofl: I've chosen `petsc_` though, as I find a prime quickly confusing with quotations.